domain/media/src/test/kotlin/com/giraffe/media/movies/usecase/GetRecentlyViewedMoviesUseCaseTest.kt

```diff
@@ -0,0 +1,40 @@
+package com.giraffe.media.movies.usecase
+
+import com.giraffe.media.movies.repository.MoviesRepository
+import com.google.common.truth.Truth.assertThat
+import io.mockk.coEvery
+import io.mockk.coVerify
+import io.mockk.mockk
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+
+
+class GetRecentlyViewedMoviesUseCaseTest {
+    private lateinit var repository: MoviesRepository
+
+    @BeforeEach
+    fun setUp() {
+        repository = mockk(relaxed = true)
+    }
+
+
+    @Test
+    fun `should returns recently watched movies`() = runTest {
```

Comment:
should return when what?
```suggestion
    fun `should returns recently watched movies when call MovieRepository`() = runTest {
```